pgdrive/scene_creator/map.py

```diff
@@ -69,6 +71,10 @@ def __init__(self, pg_world: PGWorld, map_config: dict = None):
             self.config.update(map_config)
         self.film_size = (self.config["draw_map_resolution"], self.config["draw_map_resolution"])
         self.lane_width = self.config[self.LANE_WIDTH]
+        self.lane_width_rand_range = self.config[self.LANE_WIDTH_RAND_RANGE]
+        self.lane_width = self.lane_width * random.uniform(
+            1 - self.lane_width_rand_range, 1 + self.lane_width_rand_range
```

Comment:
Is using the PGParameter class a better idea?

Beside, we have a standard way to call "uniform" function from RandomEngine, in which way the random seed can be guaranteed to be effective. Calling "random.uniform" is not a good way since the random seed may not following the same seed in Map.

Reply:
I'm on it, changing to PGParameter, thz!